emcc.py

```diff
@@ -935,11 +935,7 @@ def in_temp(name):
 
 
 def dedup_list(lst):
-  rtn = []
-  for item in lst:
-    if item not in rtn:
-      rtn.append(item)
-  return rtn
+  return list(dict.fromkeys(lst))
```

Comment:
I guess we can guarantee that this preserves ordering since we are using a recent version of python3?

Its a shame set doesn't also preserve ordering since that would be the more obvious way to implement this.

Reply:
Actually, can you add a comment regarding that?

Something like, "Since we require python 3.6, that ordering of dictionaries is guaranteed to be insertion order so we can use `dict` to here but not `set`".